docs/how-to/openstack-runner.md

```diff
@@ -8,6 +8,19 @@ enabled the charm cannot be changed to use other virtualization methods.
 
 There are three configuration that the charm needs to be deployed with to enable OpenStack integration: `openstack-clouds-yaml`, `openstack-flavor`, and `openstack-network`.
 
+## Integration
+
+The OpenStack runners require image to be available via the `image` integration. Deploy the
+`github-runner-image-builder` charm and wait for the image to be successfully provided via the
+relation data.
+
+```bash
+juju deploy github-runner-image-builder
+juju integrate github-runner-image-builder github-runner
+```
+
+The image will take about 10-15 minutes to build and be ready via the relation.
```

Comment:
Assuming this is technically correct, I think it would be more concise to say "The image will take about 10-15 minutes to build and be fully integrated." 

You also might want to tell users how to check that the deployment/integration has finished, I assume with `juju status`